statics/js/components/inject-form.js

```diff
@@ -36,6 +36,10 @@ Vue.component('inject-form', {
           <label for="inject-count">ICMP Identifier</label>\
           <input id="inject-id" type="number" class="form-control input-sm" v-model="id" min="0" />\
         </div>\
+        <div class="form-group">\
+          <label for="payload-length">Payload length</label>\
```

Comment:
I Just want to add the option to increase the size of the packets at this time (and ICMP packets are sufficient for stress testing). 

Totally agree that in future enhancements we should allow "custom payload". My thoughts are that this can be injected by allowing a user to point into something like the output of the file that one gets when capturing packets. So the custom injection can be similar to a 'reverse' of the capture process. But this is much bigger, and not part of this commit. I would wait for Sylvain to finish the new format of the capture packets process before attempting to write this 'custom injection'. For now this commit is a stand alone thing that allows us to create stress in the system and check performance ... nice addition with small amount of code.